docs/samples/transformer/image_transformer/image_transformer/__main__.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2019 kubeflow.org.
```

Comment:
Not sure if we need two files for transformer code? May be main can be combined into the 2nd file for simplicity?